src/main/java/org/black_ixx/bossshop/core/BSBuy.java

```diff
@@ -321,6 +325,35 @@ public String transformMessage(String msg, BSShop shop, Player p) {
             if (msg.contains("%priceraw%")) {
                 msg = msg.replace("%priceraw%", String.valueOf(price));
             }
+            if (msg.contains("%original_price%")) {
+                msg = msg.replace("%original_price%", MathTools.displayNumber((double) price, BSPriceType.Money));
+            }
+            if (msg.contains("%original_reward%")) {
+                msg = msg.replace("%original_reward%", MathTools.displayNumber((double) price, BSPriceType.Points));
+            }
+            if(msg.contains("%price_")) {
+                // Get the content between %price_*%,
+                String placeholder = "%price_" + msg.split("%price_")[1].split("%")[0] +"%";
+                // split on '_', check if size is == 3
+                String[] parts = placeholder.split("_");
+                if(parts.length == 3) {
+                    String shopName = parts[1];
+                    String itemId = parts[2];
+
+                    // Call the bug finder since its not implemented yet
+                    ClassManager.manager.getBugFinder()
+                            .warn(String.format("[%s:%s] We do not have support for placeholders like %s at the moment. This feature is work in progress though!", shop.getShopName(), getName(), placeholder));
+                    // Search the shop with the name and its itemId
+                    /*BSShop _shop = ClassManager.manager.getShops().getShop(shopName);
```

Comment:
Should this be here?

Reply:
Yes, this is just a small reminder for us and others that we plan to add support for that kind of placeholder. Its only printing when you try to use a placeholder like `%price_[Shop]_[Item]%`

Shall I comment it out instead?